src/components/RequestsPage/BookingRow.jsx

```diff
@@ -22,86 +25,115 @@ export default function BookingRow({
         nokButtonText: PropTypes.string,
     };
 
+    const renderFlag = (flag) => {
+        switch (flag) {
+            case "OK":
+                return <CheckCircleIcon className="h-6 w-6 text-green-noQ" />
+            case "FLAG":
+                return <FlagIcon class="h-6 w-6 text-red-700" />
+            case "DANGER":
+                return <div className="h-6 w-6 rounded-full bg-red-600"></div>;
+            default:
+                return <div className="h-6 w-6 rounded-full border-2 border-blue-700"></div>;
+        }
+    };
+
     return (
-        <div className={`grid grid-cols-[1fr_2fr_3fr_2fr_3fr_3fr_2fr_2fr_4fr] px-4 py-1 ${bg_color}`}>
-            <div className='grid grid-rows-1 justify-start self-center'>
+        <div className={`flex flex-wrap gap-x-4 px-4 py-1 ${bg_color}`}>
```

Comment:
You should use same parameters as in the RequestList component. Curretly the columns go out of sync for header and booking row. See image below.

![image](https://github.com/user-attachments/assets/948f21fa-7c4b-4990-b35b-ee2b8eda049d)